### ISSUE TITLE
Error: No mapping for source AirPlay

### ISSUE THREAD

**Issue by niXta1** [ol-iver/denonavr#40]
HA: hassio 0.62.1
AVR: Denon AVR-X2300W
Source: AirPlay

log:
"No mapping for source AirPlay
19:51 components/media_player/denonavr.py (ERROR)"

Hi!
Have I understood that I need to provide a set of mapping for my AVR?
Can someone fill me in?

**Comment by niXta1** [comment]
I tried running [the denon xml grabbing python script](https://github.com/scarface-4711/misc_python_tools/blob/master/denonavr/denon_receiver_xml.py) but only got:
```
python denon_receiver_xml.py --host 192.168.1.17 --prefix AVR-2300W
Traceback (most recent call last):
  File "denon_receiver_xml.py", line 11, in <module>
    import requests
ImportError: No module named requests
```

**Comment by ol-iver** [comment]
hi @niXta1 
it seems that you don't have installed the python _requests_ library. 
You can install it with `pip install requests`. Please try again then.

**Comment by ol-iver** [comment]
Your issue should be solved in version 0.60.
